### ISSUE TITLE
#feature request: Add 'How to use/Requirements/Intro'.

### ISSUE THREAD

**Issue by NitroNils** [genonbeta/TrebleShot#20]
I was having trouble learning about how to use this app. I learned most reading through the 'Issues' here (#15 f.x.).

Some questions were:

- How does it find devices?
-What do they need to be 'seen'?
-Do they also need the same app?
-What interfaces are utilized?
-Why aren't any devices 'seen' when I scan?
-etc.

Good luck, and thanks for the effort!

**Comment by velitasali** [comment]
@NitroNils There is a FAQ page here 
https://github.com/genonbeta/TrebleShot/blob/master/FAQ.md

**Comment by NitroNils** [comment]
Yes, thanx - but I still don't get it. Is it Android only? That would clear things up a lot to know!

**Comment by velitasali** [comment]
Hi @NitroNils,
The desktop version is in development. The Android version supports all requirements to work with the desktop version, that is why it is never explicitly stated that the app is Android only or is not. To find devices it uses Network Service Discovery that broadcasts that availability of TrebleShot. So all the devices should be running TrebleShot to share files. Network Service Discovery is available when the router supports it and most routers do. The router should also allow the local devices to interact which is something most public Wi-Fi services do not. Setting up a hotspot is also good option. You can use qr code feature to help devices discover and connect to each other.
